scapy/contrib/gtp_v2.py

```diff
@@ -266,9 +266,9 @@ class GTPHeader(gtp.GTPHeader):
                    ThreeBytesField("seq", RandShort()),
                    ConditionalField(BitField("msg_priority", 0, 4),
                                     lambda pkt:pkt.MP == 1),
-                   ConditionalField(BitField("SPARE3", 0, 4),
```

Comment:
I think that the proper way of fixing this would be something like
```python
ConditionalField(
    MultipleTypeField(
        [(BitField("SPARE3", 0, 4), lambda pkt: pkt.MP == 1)],
        ByteField("SPARE3", 0)
    ), lambda pkt: pkt.MP in [0, 1]
)
```